apps/api/src/app/layouts-v2/utils/layout-templates.ts

```diff
@@ -111,7 +111,7 @@ export const createDefaultLayout = (organizationName: string) => ({
                 {
                   type: 'text',
                   marks: [{ type: 'textStyle', attrs: { color: 'rgb(170, 170, 170)' } }],
-                  text: `${organizationName} © 2025`,
+                  text: `From ${organizationName} with ❤️`,
```

Comment:
fix for the gmail clipping; [thread](https://novu.slack.com/archives/C06DU7A7BPV/p1753777979179129)